scripts-python/create_test

```diff
@@ -135,128 +140,91 @@ formatter_class=argparse.ArgumentDefaultsHelpFormatter
     if (args.no_build):
         args.no_run = True
 
-    use_batch = cime_util.does_machine_have_batch()
```

Comment:
Can you explain why you wanted to move this stuff? It's not so bad to do argument validation in the argument parser.